components/BannerArticle.js

```diff
@@ -0,0 +1,10 @@
+import Link from "next/link";
+
+export default function BannerArticle({article}) {
+    return (
+        <Link href={`/presents/${article.slug}`}>
+            <p dangerouslySetInnerHTML={{__html: article.title.rendered}}/>
```

Comment:
I am not familiar with React's dangerouslySetInnerHTML so I looked it up. According to the following articles:

1. https://stackoverflow.com/questions/29044518/safe-alternative-to-dangerouslysetinnerhtml
2. https://blog.logrocket.com/using-dangerouslysetinnerhtml-in-a-react-application/

It seems that it makes the code vulnerable toward XSS. Will this really be safe? Or will you sanitize the DOM manipulation? What's your reasoning for using it?

Reply:
the only person that will make this vulnerable is the dumbo webstaffer that injects XSS into the wordpress article lmao